osc_sdk/sdk.py

```diff
@@ -106,16 +107,23 @@ class ApiCall(object):
     SIG_ALGORITHM = 'AWS4-HMAC-SHA256'
     SIG_TYPE = 'AWS4'
 
-    def __init__(self, **kwargs):
-        self.method = kwargs.pop('method', DEFAULT_METHOD)
-        self.access_key = kwargs.pop('access_key')
-        self.secret_key = kwargs.pop('secret_key')
-        self.version = kwargs.pop('version', DEFAULT_VERSION)
-        self.protocol = 'https' if kwargs.pop('https', None) else 'http'
-        self.region = kwargs.pop('region_name', DEFAULT_REGION)
-        self.ssl_verify = kwargs.pop('ssl_verify', SSL_VERIFY)
-        self.client_certificate = kwargs.get('client_certificate')
-        endpoint, host = (kwargs.get(x) for x in ['endpoint', 'host'])
+    def __init__(self, profile=DEFAULT_PROFILE, login=None, password=None, authentication_method=DEFAULT_AUTHENTICATION_METHOD):
```

Comment:
Unfortunatelly, it's super-long because of [Fire](https://github.com/outscale/osc-cli/pull/94/files/f67f4db58cb6fd68a29bb3934e5779eb20ea873b#diff-06ad75ae02c403423b14d840a4c84082b8972028234f6fe62bb7efb0ac922277L563) which generates help based on explicit arguments of api_connect

Reply:
What's the problem with ?
```
def __init__(self, profile=DEFAULT_PROFILE,
             login=None, password=None,
             authentication_method=DEFAULT_AUTHENTICATION_METHOD):
```

Reply:
fixed